blocks/header/header.js

```diff
@@ -54,106 +10,9 @@ function decorateOtherItems(otherItemsEl) {
-  langPicker.appendChild(langPickerUl);
-  otherItemsEl.querySelector('.lang-picker').querySelectorAll(':scope>ul>li').forEach((li) => {
-    langPickerUl.appendChild(li.cloneNode(true));
-  });
-
-  langPicker.querySelectorAll(':scope>ul>li').forEach((li) => {
-    li.classList.add('mobile-lang-picker-item');
-    li.classList.remove('lang-picker-item');
-    li.classList.remove('picker-item');
-  });
-
-  otherItemsEl.querySelector('.lang-picker').replaceWith(langPicker);
-
-  /* Move the social icons to the bottom */
-  otherItemsEl.appendChild(otherItemsEl.querySelector('.social'));
-}
-
-async function decorateLangPicker(langPicker) {
-  const lang = getLanguage() || '';
-  let langName = 'English'; // default to English
-  langPicker.classList.add('picker');
-  langPicker.classList.add('lang-picker');
-  langPicker.innerHTML = langPicker.innerHTML.replace(/\[languages\]/, '');
-
-  const currentLang = getLanguage();
-  // Get the current path without the language prefix
-  const currPath = currentLang === 'en' ? window.location.pathname : window.location.pathname.replace(`/${currentLang}/`, '/');
-  const json = await fetchIndex('query-index');
-
-  langPicker.querySelectorAll(':scope>ul>li').forEach((li) => {
-    li.classList.add('picker-item');
-    li.classList.add('lang-picker-item');
-    // Update the language links to point to the current path
-    let langRoot = li.querySelector('a').getAttribute('href');
-    langRoot = langRoot.endsWith('/') ? langRoot.slice(0, -1) : langRoot;
-    const langLink = langRoot + currPath + window.location.search;
-    li.querySelector('a').setAttribute('href', langLink);
-
-    /* Remove the current language from the list */
-    if (langRoot === `/${lang}`) {
-      langName = li.querySelector('a').innerHTML;
-      li.remove();
-    } else if (lang === 'en' && langRoot === '') {
-      // Special Check added to remove english language from the list
-      // if selected language is english
-      li.remove();
-    } else {
-      const newUrl = langRoot === '' ? `${currPath}` : `${langRoot}${currPath}`;
-      const urlExcludingSlash = newUrl.endsWith('/') ? newUrl.slice(0, -1) : newUrl;
-      const data = json.data.find((page) => [newUrl, urlExcludingSlash].includes(page.path));
-
-      if (!data) {
-        li.remove();
-      }
-    }
-  });
-
-  const a = document.createElement('a');
-  a.textContent = langName;
-  langPicker.prepend(a);
-
-  if (langPicker.querySelectorAll(':scope>ul>li').length === 0 && langPicker.querySelector('ul')) {
-    langPicker.querySelector('ul').remove();
-  }
 }
 
-function decorateTopNav(nav) {
-  nav.querySelectorAll(':scope>ul>li').forEach((li) => {
-    if (li.textContent.includes('[languages]')) {
-      decorateLangPicker(li);
-    } else if (li.textContent.includes('[websites]')) {
-      decorateWebsitePicker(li);
-    } else if (li.textContent.trim() === '[social]') {
-      decorateSocial(li);
-    }
-  });
+function decorateTopNav(/* nav */) {
```

Comment:
Do we need this?

Reply:
Right now we don't have anything in here, but it's called as part of the navbar framework so I think it would make sense to keep it, as we might need to add something in here going forward. 
I commented out the parameter to avoid the linter warning, but to remind future developers that its there.